### ISSUE TITLE
feat/create-file-from-elements

### ISSUE THREAD

**Issue by ygicp** [Unstructured-IO/unstructured#3994]
**Is your feature request related to a problem? Please describe.**
I am using the unstructured for partition a readme file. Now wherever there are images, I want to replace the images with their Alt texts. To do that I would do the following steps:
1. Partition markdown file with unstructured.
2. Detect image elements where they are present and convert them into a narrative text elements and the text of those narrative text elements should be the alt text of the image with some description.
3. I would make the markdown file back from the updated list of elements.

for creating the file back from step 3 : I do not have the function to create markdown back from list of elements. While it is relatively easier, but time taking.

**Describe the solution you'd like**
Add a function (the responsibility of the function given below) in the existing library for as many formats as you like. The responsibility of the function would be:
Re-make the entire source file from list of elements which are extracted by unstructured only.

**Describe alternatives you've considered**
I considered using regex to detect the images rather than using uns. altogether but uns. is more convenient if this feature (which I described above) is available.
